src/core/server/plugins/plugin_context.test.ts

```diff
@@ -38,6 +38,10 @@ function createPluginManifest(manifestProps: Partial<PluginManifest> = {}): Plug
     optionalPlugins: ['some-optional-dep'],
     server: true,
     ui: true,
+    owner: {
+      name: 'Core',
+      githubTeam: 'elastic/kibana-core',
+    },
```

Comment:
Should be just 'kibana-core'

Reply:
Sorry, thanks for calling that out, thought I fixed all of those.